ui/raidboss/data/04-sb/trial/byakko-ex.ts

```diff
@@ -43,17 +52,10 @@ const triggerSet: TriggerSet<Data> = {
       id: 'ByakkoEx Distant Clap',
       type: 'StartsUsing',
       netRegex: { id: '27DD', source: 'Byakko', target: 'Byakko', capture: false },
-      alertText: (_data, _matches, output) => output.text!(),
-      outputStrings: {
-        text: {
-          en: 'Distant Clap',
-          de: 'Donnergrollen',
-          fr: 'Tonnerre lointain',
-          ja: '遠雷',
-          cn: '月环',
-          ko: '원뢰',
-        },
-      },
+      condition: (data) => data.flareMarker !== data.me,
+      delaySeconds: 0.1,
+      response: Responses.getUnder(),
+      run: (data) => delete data.flareMarker,
```

Comment:
Once `flareMarker` is set the first time, it will never be deleted because of the conditional on this trigger.  The cleanup code will probably need to be in a seperate trigger that always runs.

Reply:
Wow, that's a bad oversight on my part. Sorry about that.